content/doc/book/managing/spawning-processes.adoc

```diff
@@ -0,0 +1,157 @@
+---
+layout: section
+---
+ifdef::backend-html5[]
+:notitle:
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+:hide-uri-scheme:
+endif::[]
+
+= Spawning Processes From Build
+
+Sometimes you'd like to spawn a process from a build that lives longer than the build itself. For example, maybe a part of the build is to launch a new application server with the result of the build. When you do this, you often experience a problem where the build doesn't terminate; you'll see that shell script/ant/maven terminates as expected, but Jenkins just insists on waiting, as if it didn't notice that the build is over.
+
+Starting Jenkins 1.136, Jenkins detects this situation and instead of causing infinite block, it will just print out a warning and let you get going. But you should still understand what's causing this.
```

Comment:
```suggestion
Jenkins 1.136 and later detects this situation and, instead of blocking indefinitely,  prints out a warning and terminates the build.
```

Reply:
Do we need to preserve the Jenkins version where this was first implemented?  It now sees so old that it is not relevant.